ggml/src/ggml-vulkan/vulkan-shaders/mul_mat_vec_q4_k.comp

```diff
@@ -45,55 +43,57 @@ void compute_outputs(const uint32_t first_row, const uint32_t num_rows) {
 
         [[unroll]] for (uint n = 0; n < num_rows; ++n) {
             const uint ib0 = a_offset / QUANT_K + (first_row+n)*num_blocks_per_row;
-            f16vec2 d = data_a[ib0 + i].d;
+            const f16vec2 d = data_a[ib0 + i].d;
```

Comment:
It would be nice to apply the calc_superblock changes to q4 and q5 as well, to keep all the k-quants structured the same way.